enterprise/frontend/src/embedding-sdk/components/private/PublicComponentStylesWrapper.tsx

```diff
@@ -9,15 +9,40 @@ import { saveDomImageStyles } from "metabase/visualizations/lib/save-chart-image
  * even when rendered under a React portal.
  */
 export const PublicComponentStylesWrapper = styled.div`
+  // Try to reset as much as possible to avoid css leaking from host app to our components
+  all: initial;
+  text-decoration: none;
+
+  // # Basic css reset
+  // We can't apply a global css reset as it would leak into the host app
+  // but we can't also apply our entire css reset scoped to this container,
+  // as it would be of higher specificity than some of our styles.
+  // We'll have to hand pick the css resets that we neeed
+
+  button {
+    border: 0;
+    background-color: transparent;
+  }
+  // end of RESET
+
+  font-style: normal;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+
   width: 100%;
   height: 100%;
 
   position: relative;
 
+  font-size: ${({ theme }) => theme.other.fontSize};
```

Comment:
Otherwise font leaks from the host app, try to put font-size to 1 in the host app and you'll see some text disappear in our app too without this

Reply:
We set a default value on one of the many layers we have on the theme 🫠 
https://github.com/metabase/metabase/blob/91fad45f1a6fb712c8cbe73e42680b9275be1645/enterprise/frontend/src/embedding-sdk/lib/theme/get-embedding-theme.ts#L46-L47